lib/features/selection/SelectionBehavior.js

```diff
@@ -13,10 +13,23 @@ export default function SelectionBehavior(
 
   eventBus.on('create.end', 500, function(e) {
 
-    // select the created shape after a
+    var context = e.context,
+        canExecute = context.canExecute,
+        hints = context.hints || {},
+        elements = context.elements;
+
+    // select the created shapes after a
     // successful create operation
-    if (e.context.canExecute) {
-      selection.select(e.context.shape);
+    if (canExecute) {
+
+      // check whether to be selected elements are declared
+      if (hints.selectOnFinish && hints.selectOnFinish.length) {
```

Comment:
This reminds me of AutoSelect which also accepts a hint that specifies which elements to automatically resize. The `autoResize` hint can also be `false` so no elements will be resized. I'd call the hint `autoSelect` and add the ability to set it to `false`, too.